cea/default.config

```diff
@@ -142,6 +147,32 @@ destination.help = destination folder to extract the reference case to. Since th
 # has-cooling-season = true
 # has-cooling-season.type = BooleanParameter
 
+buildings =
+buildings.type = ListParameter
+buildings.help = a list of buildings to run the simulation for - leave blank to simulate all buildings
+
+loads =
+loads.type = ListParameter
+loads.help = a list of loads to run the simulation for - leave blank to simulate all loads in demand_writers
```

Comment:
is the simulation different if not all loads are selected? (I haven't reviewed the other files, so I don't know yet, but I expect this is just a list of loads to write to the output file?)

I think `MultiChoiceParameter` would be a better choice here, as the interface in ArcGIS would then allow the user to choose them from a list - the ArcGIS interface is dynamically constructed based on the contents of the default.config file...

Reply:
good point. I just did the cahnges